addon/lib/helpers.js

```diff
@@ -1,102 +1,59 @@
 import Ember from 'ember';
-import {singularize, pluralize} from "./system/string";
+import {
+  singularize,
+  pluralize
+} from "./system/string";
 
-if (Ember.HTMLBars) {
```

Comment:
Still need this conditional.  The registerHelper imported below is ONLY for if `Ember.HTMLBars` exists.